modules/tutorials/pages/getting-started.adoc

```diff
@@ -0,0 +1,141 @@
+// suppress inspection "JsonStandardCompliance" for whole file
+
+:page-pagination:
+
+= Couchbase: Getting started
+
+:description: pass:q[A short tutorial that will guide the developer in downloading and installing Couchbase, then creating a database to store student records.]
```

Comment:
I don't think `pass` is needed here? iirc the common case is _ that would otherwise italicise text?

Reply:
fixed